php/coresky/composer.json

```diff
@@ -0,0 +1,6 @@
+{
+  "require": {
+    "php": ">7.0",
+    "energy/air": "dev-master"
```

Comment:
```suggestion
    "energy/air": "~0.501"
```

Reply:
Wait a little.. I will prepare fixed release in the packagist repository.
Let it be version: 
"energy/air": "~0.508"

Reply:
Done. Please use:
"energy/air": "0.508"